wagtail/snippets/views/snippets.py

```diff
@@ -494,6 +494,7 @@ def setup(self, request, *args, **kwargs):
         self.model_name = kwargs.get("model_name")
         self.pk = kwargs.get("pk")
         self.model = self._get_model()
+        self.object = self.get_object()
```

Comment:
Could we call this ``get_objects``/``objects`` to indicate it's a list? It also might be nice to decide whether to use ``objects`` or ``instances`` in both the view and template for consistency.

Reply:
Yeah but that means we'll need to override `post()` and `form_valid()` as well:

https://github.com/wagtail/wagtail/blob/499750735962c2a0f2b5abd32eaf8e6359490a57/wagtail/admin/views/generic/models.py#L38-L53

Would that be okay?